pkg/keyspace/keyspace.go

```diff
@@ -252,27 +267,81 @@ func (manager *Manager) saveNewKeyspace(keyspace *keyspacepb.KeyspaceMeta) error
 
 // splitKeyspaceRegion add keyspace's boundaries to region label. The corresponding
 // region will then be split by Coordinator's patrolRegion.
-func (manager *Manager) splitKeyspaceRegion(id uint32) error {
+func (manager *Manager) splitKeyspaceRegion(id uint32, waitRegionSplit bool) (err error) {
 	failpoint.Inject("skipSplitRegion", func() {
 		failpoint.Return(nil)
 	})
 
+	start := time.Now()
 	keyspaceRule := makeLabelRule(id)
-	if cl, ok := manager.cluster.(interface{ GetRegionLabeler() *labeler.RegionLabeler }); ok {
-		err := cl.GetRegionLabeler().SetLabelRule(keyspaceRule)
-		if err != nil {
-			log.Warn("[keyspace] failed to add region label for keyspace",
-				zap.Uint32("keyspaceID", id),
-				zap.Error(err),
-			)
-		}
-		log.Info("[keyspace] added region label for keyspace",
+	cl, ok := manager.cluster.(interface{ GetRegionLabeler() *labeler.RegionLabeler })
+	if !ok {
+		return errors.New("cluster does not support region label")
+	}
+	err = cl.GetRegionLabeler().SetLabelRule(keyspaceRule)
+	if err != nil {
+		log.Warn("[keyspace] failed to add region label for keyspace",
 			zap.Uint32("keyspaceID", id),
-			zap.Any("LabelRule", keyspaceRule),
+			zap.Error(err),
 		)
-		return nil
+		return err
```

Comment:
when meeting this error it will return nil in pd master, but it returns an error in pd-cse